packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -775,15 +784,24 @@ export default {
   border-radius: var(--dt-size-radius-400);
   border: var(--dt-size-border-100) solid;
   border-color: var(--dt-color-border-default);
+  box-shadow: 0 0 0 0 rgba(0, 0, 0, 0%);
```

Comment:
Is there an appropriate token for this? isn't this just removing the box-shadow? If so, it's not the same as just `box-shadow: none;`?

Reply:
It sets the box shadow to a 0% opacity for animation/transition purposes. Doing it this way inserts the box-shadow into the DOM but hides it.